fenix/app/src/main/java/org/mozilla/fenix/settings/search/Extensions.kt

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.search
+
+import mozilla.components.browser.state.search.SearchEngine
+import mozilla.components.browser.state.state.SearchState
+import mozilla.components.browser.state.state.availableSearchEngines
+
+/**
+ * The list of search engine shortcuts to be available for quick search menu.
+ */
+val SearchState.searchEngineShortcuts: List<SearchEngine>
+    get() = (
+        regionSearchEngines + additionalSearchEngines + availableSearchEngines +
+            customSearchEngines + applicationSearchEngines
+        ).filter {
+        !disabledSearchEngineIds.contains(it.id)
+    }
```

Comment:
Moving the extension function to A-C wouldn't prevent Focus from defining their own ordering, and I think A-C already has some opinionated functionality exposed in terms of ordering, like in `SearchState.searchEngines`. 

I also see that there is already new code in A-C that references `shortcutIds` like `BrowserState.updateDisabledSearchEngineShortcutsIds` and `class UpdateDisabledSearchEngineShortcutsIdsUseCase`, which is another part of my reasoning.

That said, I'm not overly concerned either way so I'll leave it up to whatever you want to do.

Reply:
Actually, keeping those was an overlook on my part, as @jonalmeida asked to get rid of the shortcut name.
Since Jonathan originally asked me to move the extension away from the `SearchState`, I wonder if he could share his opinion on the subject :)

Reply:
moved to a separate file, but kept inside Fenix